runtime/compiler/control/J9Options.cpp

```diff
@@ -1723,114 +1720,47 @@ J9::Options::fePreProcess(void * base)
                   j9nls_printf(PORTLIB, J9NLS_INFO, J9NLS_JIT_OPTIONS_LARGE_PAGE_SIZE_NOT_SUPPORTED_WITH_PAGETYPE, oldSize, oldQualifier, oldPageType, newSize, newQualifier, newPageType);
                }
             }
-
-         if (largeCodePageSize > 0)
-            {
-            jitConfig->largeCodePageSize = largeCodePageSize;
-            jitConfig->largeCodePageFlags = largeCodePageFlags;
-            }
          }
+      // No <size> for -Xlp or -Xlp:codecache, default (and -Xlp) behavior is to use preferred page size
```

Comment:
-Xlp:codecache has to specify a page size, there seems to be code above that gives an error if isn't not specified.

Reply:
That's a poorly written comment. 

Will replace with "When no -Xlp arguments are passed, we should use preferred page sizes"